JSDemos/Demos/Gauges/Overview/Angular/description.md

```diff
@@ -1 +1 @@
-DevExtreme Angular Gauges are interactive UI components for dashboards. They come in two geometric shapes - circular, or radial, and linear - and provide a comprehensive set of features including multiple value indicators, geometry customization, interactive tooltips, palettes support, and many more. Native Angular features, like AOT compilation, declarative configuration, TypeScript compile-time checking, are supported as well.
\ No newline at end of file
+DevExtreme Angular Gauges are interactive UI components for dashboards. They come in two geometric shapes - circular, or radial, and linear - and provide a comprehensive set of features including multiple value indicators, geometry customization, interactive tooltips, palettes support, custom labels in the component's center, and many more. Native Angular features, like AOT compilation, declarative configuration, TypeScript compile-time checking, are supported as well.
```

Comment:
I explain the changes in the jQuery description (making descriptions consistent): 

```suggestion
DevExtreme Angular Gauge controls help you visualize data and create dashboards. Our library includes three gauge types: radial bar gauges, circular scale gauges, and linear gauges. 

This demo displays a simple dashboard with a few highly customized gauges. 

Review other demos in this section to explore gauge features such as geometry customization, palettes, tooltips, and custom label display.

Our gauge controls support Native Angular features as well: AOT compilation, declarative configuration, and TypeScript compile-time checking.
```